scm/driver/gitlab/webhook.go

```diff
@@ -904,4 +964,139 @@ type (
 			Homepage    string `json:"homepage"`
 		} `json:"repository"`
 	}
+
+	pipelineHook struct {
+		ObjectKind       string           `json:"object_kind"`
+		ObjectAttributes ObjectAttributes `json:"object_attributes"`
+		MergeRequest     MergeRequest     `json:"merge_request"`
+		User             user             `json:"user"`
+		Project          Project          `json:"project"`
+		Commit           Commit           `json:"commit"`
+		SourcePipeline   SourcePipeline   `json:"source_pipeline"`
+		Builds           []Build          `json:"builds"`
+	}
+
+	ObjectAttributes struct {
```

Comment:
all of these structs should be lowercase, as there is no need to expose these as public structs

Reply:
Updated the struct